### ISSUE TITLE
No tags matching UPBGE/upbge tags

### ISSUE THREAD

**Issue by david-jennings** [UPBGE/blender-addons#7]
[UPBGE/upbge](https://github.com/UPBGE/upbge) has tags that loosely correspond to the tags for Blender releases. However this repository doesn't have tags making it difficult to get matching versions.

Specifically, we would like to use the latest stable - [upbge 0.36.1](https://github.com/UPBGE/upbge/releases/tag/v0.36.1) - but we then get an error with blender-addons attempting to load AssetShelf, as introduced in the post 0.36.1 commit [2faef810](https://github.com/UPBGE/blender-addons/commit/2faef810411b17d50b28b42eea5e1bc18f9f98f2).

We will try to work-around this issue by downloading the commit that seems to match the [blender/blender-addons](https://github.com/blender/blender-addons/tree/v3.6.2) tag but I'm worried that this might miss UPBGE fixes to any incompatibilities introduced by that merge.

Is it possible to have tags on this repo?

**Comment by lordloki** [comment]
Yes, It was my fault. For next releases I will tag this repository when I do the release similar to upbge repository.
Currently, to mitigate the issue with the addons you can download blander 3.6.2 and copy the pose library folder (blender-3.6.2-windows-x64\3.6\scripts\addons\pose_library) into the similar one on upbge installation.

**Comment by david-jennings** [comment]
Thanks. That'd be great. In the meantime I'm testing using commit ee1f3fe4 of UPBGE/blender-addons, which seems to match [3.6.2 from blender/blender-addons](https://github.com/blender/blender-addons/tree/v3.6.2).
